OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -267,14 +267,15 @@ bool HandleLeftClick(ProductionItem item, ProductionIcon icon, int handleCount,
 		{
 			if (PickUpCompletedBuildingIcon(icon, item))
 			{
-				Game.Sound.Play(SoundType.UI, TabClick);
+				Game.Sound.PlayNotification(World.Map.Rules, World.LocalPlayer, "Sounds", ClickSound, null);
```

Comment:
Maybe convert the "Sounds" to a const and use it on all the places

Reply:
"Sounds" is used across 17 files, so would rather not do that in this PR